content/guides/hmr-react.md

```diff
@@ -157,6 +157,21 @@ module.exports = {
 };
 ```
 
+Important to note:
+
+If you have a `query` section for the `babel-loader` in your `rules`, remember to turn off the `modules` tranformation as well, otherwise HMR won't work.
+
+```js
+loader: "babel-loader",
+query: {
```

Comment:
Hi, Thanks for reading @skipjack 
I read that [part](https://webpack.js.org/guides/hmr-react/#babel-config) before. The whole document is very clear and smooth.

But it is good for a new user but not an existing config. That's why it cost me some time to figure it out. Yes, it's a mistake by the user rather than the documents. But it would be better if it covers that part.

But I think your suggestion does make sense. If you guys think this still needs to be added, I could do a new commit. :)